package.json

```diff
@@ -120,12 +124,11 @@
     "get-port": "^3.2.0",
     "glob": "^7.1.6",
     "graphql": "0.13.2",
-    "int64-buffer": "^0.1.9",
+    "gzip-js": "^0.3.2",
```

Comment:
Similar to pako, why is this needed?

Reply:
This is leftover from some refactor and will be removed